src/Temporalio/Worker/WorkflowInstance.cs

```diff
@@ -2452,6 +2490,106 @@ public override Task<ChildWorkflowHandle<TWorkflow, TResult>> StartChildWorkflow
                 return handleSource.Task;
             }
 
+            /// <inheritdoc/>
+            public override Task<NexusOperationHandle<TResult>> StartNexusOperationAsync<TResult>(
+                StartNexusOperationInput input)
+            {
+                var token = input.Options.CancellationToken ?? instance.CancellationToken;
+                // We do not even want to schedule if the cancellation token is already cancelled.
+                // We choose to use cancelled failure instead of wrapping in child failure which is
```

Comment:
```suggestion
                // We choose to use cancelled failure instead of wrapping in nexus operation failure which is
```

I think this is what you mean?

Reply:
Yes, this is a typo via copy pasta from child method above

Reply:
Fixed